decidim-proposals/app/views/decidim/proposals/admin/proposals/index.html.erb

```diff
@@ -40,17 +40,14 @@
                 <% end %>
               </td>
               <td>
-                <% if proposal.state.nil? %>
-                  <% if can? :update, proposal %>
-                    <%= link_to t("actions.answer", scope: "decidim.proposals"), edit_proposal_proposal_answer_path(proposal_id: proposal.id, id: proposal.id), class: "button tiny" %>
-                  <% end %>
-                <% else %>
-                  <strong class="<%= proposal.state == 'accepted' ? 'text-success' : 'text-alert' %>">
-                    <%= humanize_proposal_state proposal.state %>
-                  </strong>
-                <% end %>
+                <strong class="<%= proposal_state_css_class proposal.state %>">
+                  <%= humanize_proposal_state proposal.state %>
```

Comment:
I know this isn't strictly from this PR, but I think you can accept a proposal without answering it. If this is true, saying "Not answered" when a proposal is not evaluated feels weird, as it can be not answered and evaluated at the same time.

I think it would be clearer if we said "Not evaluated" instead of "Not answered" when a proposal is not accepted/rejected.